bubblewrap.c

```diff
@@ -1458,6 +1485,9 @@ main (int argc,
   /* We want sigchild in the child */
   unblock_sigchild ();
 
+  if (label_exec(opt_exec_label) == -1)
```

Comment:
It's an interesting topic...presumably *not* relabeling it will mean that the child processes would have restricted access to pid 1 (or we could implement that).  Maybe that'd be useful for something?  Dunno.  Can't think of anything offhand.